app/components/TipsDetails.tsx

```diff
@@ -63,6 +63,10 @@ export const TipsDetails = ({ schoolName, canEdit }: { schoolName: string; canEd
 
   // Function to handle deleting a tip
   const handleDelete = async (tipID: string) => {
+    if (!userInfo?.isAdmin) {
```

Comment:
Probably don't need this because if the button doesn't exist then they can never trigger this function